clang/lib/CIR/FrontendAction/CIRGenAction.cpp

```diff
@@ -470,3 +472,51 @@ EmitBCAction::EmitBCAction(mlir::MLIRContext *_MLIRContext)
 void EmitObjAction::anchor() {}
 EmitObjAction::EmitObjAction(mlir::MLIRContext *_MLIRContext)
     : CIRGenAction(OutputType::EmitObj, _MLIRContext) {}
+} // namespace cir
+
```

Comment:
Same comment here: `// Used for -fclangir-analysis-only: use CIR analysis but still use original LLVM codegen path`

Reply:
Done